app/lib/pre_assembly/reporting_old.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: UTF-8
 
 module PreAssembly
-  module Reporting
+  module ReportingOld
```

Comment:
I think this change unintentionally crept in

Reply:
actually, you reviewed the PR before I hit save on my edits to comment above. 

>Note that the edit to reporting_old class helped me test this on the console on -stage; I know the class is going away.

essentially, without this stupid edit, it has two modules called the same thing so I couldn't execute the one I wanted when manually testing that I had all the settings correct and accounted for.

Reply:
Ah, that makes sense.  I forgot you already copied that class.